### ISSUE TITLE
Renataki's Charm of Beasts

### ISSUE THREAD

**Issue by Phenomenal101** [dalaranwow/dalaran-wow#5955]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
https://youtu.be/wMAbj2fceeo : as can be seen here, the trinket does not reset the cooldown on Aimed Shot and Multi-Shot like it's supposed to. Volley does not have a CD anymore in WotLK so it does not have to remove that cooldown.

**Expected behaviour**: Tell us what should happen instead
https://youtu.be/4ui6mf3lHSU?t=50 : this is what it's supposed to do, as can be seen in the video it instantly clears the cooldown on Multi-Shot even right after using it.

**Steps to reproduce the problem**:

1. No steps needed, the trinket takes a lot of time to obtain in Zul'Gurub. The trinket does not what it's meant to do.
2. -
3. -

**Include proofs for this behaviour**

Proof shown in the form of YouTube video's above.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

https://www.wowhead.com/item=19953/renatakis-charm-of-beasts
ID: 133306

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
